readthedocs/builds/admin.py

```diff
@@ -13,14 +13,22 @@ class BuildCommandResultInline(admin.TabularInline):
 
 class BuildAdmin(admin.ModelAdmin):
     fields = ('project', 'version', 'type', 'state', 'error', 'success', 'length', 'cold_storage')
-    list_display = ('project', 'success', 'type', 'state', 'date')
+    list_display = ('id', 'project', 'version_name', 'success', 'type', 'state', 'date')
```

Comment:
nit: I think the `id` could be removed from this list since I suppose it's not very useful.

Reply:
Oh! If we remove the `id` the link would be the `project` and it will be confusing. So, I suppose that was what you wanted. In that case, I'm :+1: on leaving it.

I'm sorry.